### ISSUE TITLE
Invalid Expires header date triggers decode exception

### ISSUE THREAD

**Issue by d6y** [Spinoco/protocol#8]
## Observation

Using 0.1.7 to make a http cliient request to the Slack API server I see:

```
java.lang.Throwable: Failed to decode http response :Headers/Expires: Invalid format :  Mon, 26 Jul 1997 05:00:00 GMT : For input string: "mon, 26 jul 1997 05:00:00 gmt"
	at spinoco.fs2.http.HttpResponse$$anonfun$fromStream$2$$anonfun$apply$10.apply(HttpRequestOrResponse.scala:310)
	at spinoco.fs2.http.HttpResponse$$anonfun$fromStream$2$$anonfun$apply$10.apply(HttpRequestOrResponse.scala:308)
	at fs2.Stream$$anonfun$flatMap$1.apply(Stream.scala:151)
  ...etc...
```

## Investigation

`Mon, 26 Jul 1997 05:00:00 GMT` is not a valid date in the sense that 26 Jul 1997 was a Saturday.
However, the [HTTP Spec](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.21) allows for this:

>  HTTP/1.1 clients and caches MUST treat other invalid date formats, especially including the value "0", as in the past (i.e., "already expired"). 

## Expected Behaviour

Hmm.... not an exception. I'm not making use of the expires header, so don't have an opinion on what the type of the value should be.

**Comment by pchlupacek** [comment]
@d6y thanks for reporting that. I think we can encode it as Expires(Left(0)) ? WDYT?

**Comment by d6y** [comment]
@pchlupacek yes - makes sense to me.

**Comment by pchlupacek** [comment]
actually solved in 0.2.1 thanks for reporting this.

**Comment by d6y** [comment]
Thank you!
